controllers/rhmi/bootstrapReconciler.go

```diff
@@ -612,26 +612,25 @@ func (r *Reconciler) reconcileAddonManagedApiServiceParameters(ctx context.Conte
 }
 
 func (r *Reconciler) retrieveConsoleURLAndSubdomain(ctx context.Context, serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
-
+	if r.installation.Spec.RoutingSubdomain != "" {
+		log.Info("routing subdomain already set, exiting retrieveConsoleURLAndSubdomain() early")
+		return integreatlyv1alpha1.PhaseCompleted, nil
+	}
```

Comment:
I'm not too sure about this :thinking:  Would this not prevent using the custom domain work if the customer decides to use that at a later stage after the intial install ? :thinking:

Reply:
This was brought from rebase with master. It was decided that custom domain can only be specified before the RHOAM installation has begun

Reply:
Oh cool, thanks :+1: